.travis.yml

```diff
@@ -2,6 +2,12 @@ language: rust
 sudo: required
 dist: xenial
 
+branches:
+  only:
+  - auto
+  - try
```

Comment:
What is this branch? We don't currently seem to have a branch with this name.

Reply:
It's used when you do `@bors try`.